packages/menu/src/types.ts

```diff
@@ -86,8 +86,19 @@ export interface IUseMenuProps<T = HTMLButtonElement, M = HTMLElement> {
     focusedValue?: string | null;
     selectedItems?: ISelectedItem[];
   }) => void;
-  /** Sets the environment where the menu is rendered */
+  /**
+   * Sets the environment where the menu is rendered
+   * @deprecated use `window` prop instead.
+   * */
   environment?: Window;
```

Comment:
Let's not deprecate; prefer the immediate `feat(menu)!: ...` breaking change in `react-containers`